Assets/Scripts/Meteor/MeteorMove.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MeteorMove : MonoBehaviour
+{
+    [SerializeField]
+    private float m_Speed;
+    [SerializeField]
+    private GameObject m_ImpactPrefab;
+    [SerializeField]
+    private Transform m_EndPoint;
+    [SerializeField]
+    private Transform m_StartPoint;
+
+    private void Update()
+    {
+        transform.position = Vector3.Lerp(m_StartPoint.position, m_EndPoint.position, 2.5f);
```

Comment:
A little confused by this line. Doesn't this give the meteor the same position every call to update?

Reply:
This looks like a bug. Remind me to talk about the `Lerp` function during our weekly meeting.

Reply:
I am not sure how to perform Lerp function but I've stopped using this script, replace with animations.